packages/perseus/src/widgets/graded-group-set/graded-group-set.test.ts

```diff
@@ -312,4 +312,38 @@ describe("graded group widget", () => {
             screen.queryByRole("button", {name: "Hide explanation"}),
         ).not.toBeInTheDocument();
     });
+
+    it("should show rationales when answer is correct", async () => {
```

Comment:
We can go with this for now, but I feel this is a big regression in usefulness of the graded-group (and graded-group-set) widget.  These are used mostly in articles (and I think in hints).  So only showing rationales once the learner has gotten the correct answer feels like it doesn't help learning. 

This probably needs separate design work, but I'd think both of these group widgets need some affordance for the learner to "skip" and see the rationales.

Reply:
I definitely agree! The user is still told if they're correct/incorrect with the graded-group's correctness indicator, but I also felt that we should add a skip button to the widget to allow students to quickly see the full rationales. 

It wouldn't be hard to add, but I figured this PR is already casting a pretty wide net. :) I can make a ticket for us to revisit the topic as a potential follow up